armi/reactor/tests/test_components.py

```diff
@@ -339,13 +339,14 @@ def test_getBoundingCircleOuterDiameter(self):
 
 class TestCircle(TestShapedComponent):
     componentCls = Circle
+    _id = 5.0
     _od = 10
     _coldTemp = 25.0
     componentDims = {
         "Tinput": _coldTemp,
         "Thot": 25.0,
         "od": _od,
-        "id": 5.0,
+        "id": _id,
         "mult": 1.5,
     }
```

Comment:
I love the tests!

Can we move this stuff into a `def setUp()` method?  Take a look around ARMI, we use them in many/most test files.

Just to make things a bit more consistent across the repo.

Thanks!

Reply:
Happy to. This was just how the tests were set up when I got there. Many of the other test classes in this file do not use the `def setUp()` approach and just define class variables. Do you want me to take a stab at that too? If so, maybe it would be better to have it in a separate PR? Just to keep changes separated a little.

Reply:
Nevermind, if it's in lots of places, I will fix them all in one swoop later.